MANIFEST

```diff
@@ -2,10 +2,10 @@
 Changes
 docker/Dockerfile
 docker/zonemaster_startup.sh
+docs/FAQ/gui-faq-da.md
 docs/FAQ/gui-faq-en.md
 docs/FAQ/gui-faq-fr.md
 docs/FAQ/gui-faq-sv.md
-docs/FAQ/gui-faq-da.md
```

Comment:
You must not remove this!

Reply:
I didn't. See line 5 after the update. When you run `make manifest` you get a lexicographically ordered MANIFEST file.